pkgs/development/python-modules/opshin/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, setuptools
+, poetry-core
+, pythonRelaxDepsHook
+# Python deps
+, uplc
+, pluthon
+, frozenlist2
+, pycardano
+, astunparse
+, ordered-set
+}:
+
+buildPythonPackage rec {
```

Comment:
Should this be using `buildPythonApplication` instead of `buildPythonPackage`?

Reply:
Yeah, this one probably should. I guess it also should be moved out of `python-modules` right?